modules/tensor_mechanics/test/tests/beam/action/tests

```diff
@@ -1,109 +1,174 @@
 [Tests]
+  design = "LineElementMaster/index.md"
+  issues = "#10313"
   [./2_block_action]
     type = 'Exodiff'
     input = '2_block.i'
     exodiff = '2_block_out.e'
+
```

Comment:
@aeslaughter Does it work if we split these across multiple lines with quotes around them like this (and is that acceptable formatting)?

Reply:
We talked about this, and we can do this either way at this point. There's an input file formatter in the works that will reformat these all according to established style guides pretty soon anyway, so no need to worry about this.